CHANGELOG.md

```diff
@@ -146,13 +146,13 @@ Adds startmsg_regex and endmsg_regex options:
 
 - Fix a bash bug in changelog_to_tag.yml, which unexpectedly expanded "*"
 
-- changelog_to_tag action - support other than "master" for the main branch name, as well
+- changelog_to_tag action - support other than "master" for the main branch name, as well // wokeignore:rule=master
 
 - Use GITHUB_REF_NAME as name of push branch; fix error in branch detection [citest skip]
 
 We need to get the name of the branch to which CHANGELOG.md was pushed.
 For now, it looks as though `GITHUB_REF_NAME` is that name.  But don't
-trust it - first, check that it is `main` or `master`.  If not, then use
+trust it - first, check that it is `main` or `master`.  If not, then use // wokeignore:rule=master
```

Comment:
ok - but I'd like to see what @Jakuje says about https://github.com/linux-system-roles/crypto_policies/pull/54#issuecomment-1352004655 - if the right answer is "just edit CHANGELOG.md to remove the problematic terms rather than put `wokeignore` in CHANGELOG.md" then this PR becomes much smaller and simpler i.e. can use the official woke instead of the custom

Reply:
Yeah, but this changelog item specifically refers to the `master` branch... How we could describe it without using the word `master`... In this case, I can just remove it, as it does not look that important(?), but there could be some cases that do not allow it???

Reply:
> Yeah, but this changelog item specifically refers to the `master` branch... How we could describe it without using the word `master`... In this case, I can just remove it, as it does not look that important(?), but there could be some cases that do not allow it???

I think we can remove these entire entries from the changelog.

The problem with the auto-changelog generation that I wrote is
* it is pretty stupid - it just collects all of the git commit messages
* it requires a non-lazy human to edit - I should really have just summarized all of these changes as something like "fix github publish action" or something like that

Especially in this case - these can be replaced with `- fix github action for publishing roles` or similar

Reply:
Same below - just say `- github action ansible test improvements`

Reply:
Thanks, @richm! I'm updating CHANGELOG as you suggested.